src/components/LearningResourcesWidget/LearningResourcesWidget.scss

```diff
@@ -0,0 +1,3 @@
+.widget-learning resources {
+  column-width: 300px;
```

Comment:
Not sure where this is being used, but do we want the pixel value here? Something relative might work better